mcgill-en.csl

```diff
@@ -38,13 +41,20 @@
       <term name="ordinal-11">th</term>
       <term name="ordinal-12">th</term>
       <term name="ordinal-13">th</term>
+	  <term name="open-quote">"</term>
```

Comment:
actual straight quotes? That seems wrong. I don't have access to the guide, but the samples at Calgary and Carleton both have regular (typographic) quotes.

Reply:
Hi, and yes - I should have removed this before finalizing.  Will remove.